src/pages/Favorites/FavoritedIcon.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="15" 
```

Comment:
oo if we add this directly into the component we might not need separate svg

Reply:
I tend to agree if we have to pass in some props that svgs can't take in (like completed/favorite button), but I'm thinking this will be cleaner to edit in future since its embedded in Typography message